ui-service/src/main/frontend/hotel-booking-ui/src/components/PromotionComponent/PromotionComponent.tsx

```diff
@@ -0,0 +1,168 @@
+import { BsQrCodeScan } from "react-icons/bs";
+import { BsSuitcaseLg } from "react-icons/bs";
+import { IoHelpCircleOutline } from "react-icons/io5";
+import { FaChevronRight } from "react-icons/fa6";
+import { ReactElement, useState } from "react";
+import { MdOutlineThumbUp } from "react-icons/md";
+
+const PromotionComponent = () => {
+  return (
+    <div className="flex flex-row max-w-6xl mx-auto px-6 py-8 gap-4 items-start">
+      <AppDownloadSection />
+      <div className="flex flex-col gap-4">
+        <FindMyTripSection />
+        <HelpCenterSection />
+      </div>
+    </div>
+  );
+};
+
+export default PromotionComponent;
+
+const AppDownloadSection = () => {
+  const [phoneNumber, setPhoneNumber] = useState("");
+  const [linkSent, setLinkSent] = useState(false);
+
+  const formatPhoneNumber = (input: string) => {
+    // Prepend "+1 " for the first entered digit
+    if (!input.startsWith("+")) {
+      input = "+1 " + input;
+    }
+
+    // Remove all non-digit characters
+    const cleaned = input.replace(/\D/g, "");
+
+    // Limit to 10 digits
```

Comment:
If the phone number is 10 digits and you prepend 1, isn’t the total should be 11 digits

Reply:
Yeah, it should be 11 digits.